### ISSUE TITLE
Unmarshal Error for GetChatMenuButton

### ISSUE THREAD

**Issue by xtianpu** [mr-linch/go-tg#123]
Hello! I got an error like below when I call Do() of GetChatMenuButton():

```
panic: unmarshal: json: cannot unmarshal object into Go value of type tg.MenuButton [recovered]
```

**Comment by mr-linch** [comment]
Hey, thanks for the issue! Problem solved, now I will look at the second problem you found and do a release.
Feel free to reopen if you have any problems related to that method.

**Comment by mr-linch** [comment]
@xtianpu that fix was released in [v0.9.1](https://github.com/mr-linch/go-tg/releases/tag/v0.9.1). 

FYI: https://github.com/mr-linch/go-tg/issues/124#issuecomment-1557735631
